### DISCUSSION TITLE
60fps for nikoderico

### DISCUSSION THREAD

**Discussion by rakantoh** [StevensND/switch-port-mods#25] [Mod Request]
as title: 60fps for nikoderiko, thanks.

**Comment by rakantoh** [comment]
nevermind, i already found 60fps and ultrawide mod for the game and version 1.0.1, so i close the request.

**Comment by StevensND** [comment]
[Done by Fl4sh](https://github.com/Fl4sh9174/Switch-Ultrawide-Mods/blob/main/Nikoderiko%20the%20Magical%20World%20%5B21.9%20Ultrawide%20v1.0.1%5D.zip)

I've been very busy these days due to work and I couldn't mod until today.
